parse_proto.py

```diff
@@ -279,7 +303,7 @@ def __init__(self, pos, modifier, ftype, name, index, default=None):
         self.type = ftype
         self.name = name
         self.index = int(index)
-        self.default = default
+        self.default = True if default == 'true' else default
```

Comment:
The default value for bools in protobuf is false so in practice I don't think it would explicitly set as a default, but it should be handled correctly here in case it is.